roles/towerdeploy/meta/main.yml

```diff
@@ -0,0 +1,17 @@
+---
+galaxy_info:
+  role_name: towerdeploy
+  company: Princeton University Library
+  description: deploy from ansible tower
+  author: pulibrary
+
+  license: MIT
+
+  min_ansible_version: 2.9
+
+  platforms:
+    - name: Ubuntu
+      versions:
+        - jammy
+dependencies:
```

Comment:
Can we remove this dependency? Otherwise it will run the ruby_s role twice, I think - my understanding is that this was the purpose of having the roles in the playbook, rather than the meta.yml.

Reply:
we will be unable to test this role if we don't have `ruby_s`